site/content/en/docs/Reference/fleetautoscaler.md

```diff
@@ -37,8 +37,6 @@ spec:
       # maximum fleet size that can be set by this FleetAutoscaler
       # required
       maxReplicas: 20
-  # [Stage:Beta]
```

Comment:
This update should also be feature shortcoded please.

Reply:
I have a little doubt about this update. Is this correct?
  {{% feature expiryVersion="1.34.0" %}}
  # [Stage:Beta]
  # [FeatureFlag:CustomFasSyncInterval]
  {{% /feature %}}

Reply:
Thaaat might work? But I'm betting it won't.

If you run `make site-server` you can preview your changes locally and see what works and what doesn't.

Reply:
Is the behavior shown in the screenshot at https://screenshot.googleplex.com/5nzmXVU6eacV8Kv expected?

There was an issue with `make site-server` command. This problem has been resolved by adding `git config --global --add safe.directory /go/src/agones.dev/agones` to the existing code.

Reply:
Yeah, that doesn't look right (it should be gone), you can also see the preview here: https://1b8d8e1-dot-preview-dot-agones-images.appspot.com/site/docs/reference/fleetautoscaler/

I expect you'll need to feature code the entire block.

Reply:
I haven't observed any variation in my local environment, whether I utilize feature shortcodes or not. Even if I remove the entire block, I still encounter these lines:
```
# [Stage:Beta]
# [FeatureFlag:CustomFasSyncInterval]
```

Reply:
I think I wasn't clear - much like we do with markdown tables, we would have to duplicate the whole

```
```yaml
....
...
```

block.

Reply:
I tried this too and it was not working in my local 😞 There might be some issue with my local env. committed the changes..